tests/vmi_configuration_test.go

```diff
@@ -1732,6 +1735,23 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 				Expect(origVmiWithoutHeadroom.Spec.Domain.Memory).To(Equal(vmiWithoutHeadroom.Spec.Domain.Memory), "vmi guest memory is not expected to change")
 			})
 		})
+
+		It("be able to use hyperv passthrough and ensure it's non-migratable", func() {
+			vmi := libvmi.NewFedora(libvmi.WithHypervPassthrough())
+			vmi = tests.RunVMIAndExpectLaunch(vmi, 60)
+
+			domXml, err := tests.GetRunningVirtualMachineInstanceDomainXML(virtClient, vmi)
+			Expect(err).ToNot(HaveOccurred())
+
+			domSpec := &api.DomainSpec{}
+			Expect(xml.Unmarshal([]byte(domXml), domSpec)).To(Succeed())
+			Expect(domSpec.Features.Hyperv.Mode).To(Equal(api.HypervModePassthrough))
+
+			conditionManager := controller.NewVirtualMachineInstanceConditionManager()
+			cond := conditionManager.GetCondition(vmi, v1.VirtualMachineInstanceIsMigratable)
+			Expect(cond).ToNot(BeNil())
+			Expect(cond.Status).To(Equal(kubev1.ConditionFalse))
```

Comment:
This can be flaky:
```suggestion

			Eventually(matcher.ThisVMI(vmi), 60*time.Second, 1*time.Second).Should(matcher.HaveConditionFalse(v1.VirtualMachineInstanceIsMigratable))
```

Reply:
Awesome! Thanks a lot!
done